src/robot/conf/languages.py

```diff
@@ -1356,3 +1356,47 @@ class Ko(Language):
     but_prefixes = ['하지만']
     true_strings = ['참', '네', '켜기']
     false_strings = ['거짓', '아니오', '끄기']
+
+class Ar(Language):
+    """Arabic
+
+    New in Robot Framework X.Y.
```

Comment:
You can change `X.Y` this to `7.3` because that's the next release (unless surprises happen).

Please also add second empty line before the `class Ar(Language):` line to follow our coding conventions. Adding a newline at the end of the file would be nice too.